website/pages/docs/builders/vmware/vsphere-iso.mdx

```diff
@@ -41,6 +41,19 @@ necessary for this build to succeed and can be found further down the page.
 
 @include 'builder/vsphere/iso/Config-not-required.mdx'
 
+### Boot configuration
```

Comment:
I remember adding this when I refactored the boot_command code. 

It already exists bellow in the doc here, somewhere near the line 175: 
https://5f046f0a96c3d0000738f89c--packer-www.netlify.app/docs/builders/vmware/vsphere-iso#boot-configuration-1

Reply:
Ah thanks for pointing that out. I'll keep your version and just update with the new partial.